addon/components/base-layer.js

```diff
@@ -7,7 +7,13 @@ const { assert, computed, Component, run, K, A, String: { classify } } = Ember;
 
 export default Component.extend(ChildMixin, InvokeActionMixin, {
   tagName: '',
-  L,
+
+  //FASTBOOT-TODO: Exposing L here is a problem
+  L: Ember.computed('window.L', function(){
+    if(L){
```

Comment:
I don't see how this computed property would help.
As I understand it, this CP returns undefined if `L` is undefined.
Isn't that exactly the same as doing `L: L` in the object definition?

Also, depending on `window.L` doesn't make a difference. Ember will try to look up a `window` object in the BaseLayer context.